src/octopeer-github/main/Status.ts

```diff
@@ -90,27 +90,11 @@ const Status = new (class Status {
      */
     public set(status: StatusCode) {
         this.status = status;
-        if (Options.get(Options.LOGGING)) {
-            this.setter(status);
-        } else {
-            this.setter(StatusCode.OFF);
-        }
-    }
 
-    /**
-     * Accepts a status and returns true if this is the current status.
-     * @param s the status to check against
-     * @returns {boolean} whether the current status equals s
-     */
-    public isStatus(s: StatusCode): Boolean {
-        return s === this.status;
-    }
+        if (!Options.get(Options.LOGGING)) {
+            status = StatusCode.OFF;
+        }
```

Comment:
I know this was already the case before the refactor, but can anyone explain why the internal status is not set to `OFF` when `LOGGING` is false?